src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -505,7 +505,7 @@ public boolean isAllowed(CaseDetails caseDetails, CaseEvent caseEvent) {
 
         if ((caseData.getSuperClaimType() != null && caseData.getSuperClaimType().equals(SPEC_CLAIM))
             || caseEvent.toString().equals("CREATE_CLAIM_SPEC")) {
-            StateFlow stateFlow = stateFlowEngine.evaluateSpec(caseDetails);
+            StateFlow stateFlow = stateFlowEngine.evaluate(caseDetails);
```

Comment:
I believe this line can now move out of if condition, as same method is being called in both Claim types. We can check the condition to call isAllowed... method only.

Reply:
true, I have removed conditions